src/main/java/com/exactpro/th2/inframgr/k8s/K8sOperator.java

```diff
@@ -72,30 +73,31 @@ private void startInformers() {
         }
 
         logger.info("Creating informers");
-        Kubernetes kube = new Kubernetes(config.getBehaviour(), config.getKubernetes(), null);
-        cache = K8sResourceCache.INSTANCE;
+        try (Kubernetes kube = new Kubernetes(config.getBehaviour(), config.getKubernetes(), null)) {
```

Comment:
Honestly, it looks like this class should have the kubernetes instance as a field